coreapp-dev/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM        apiaryio/base-dev-nodejs
 MAINTAINER  Apiary <sre@apiary.io>
 
-ENV REFRESHED_AT 2010-02-05
+ENV REFRESHED_AT 2010-02-10
```

Comment:
These images are built automatically, so whenever there is a push to repo they get rebuilt, don't they? So there is no need to change the file to trigger the build, am I correct?